readme/syncable_units.md

```diff
@@ -0,0 +1,11 @@
+### Model `syncable_units`
+
+The `syncable_units` subfolder allows to import syncable units provided as CSV files. See the example below:
+
+| id | name   
+| - | - | - |
+| init.syncable_unit_1 | UI 
```

Comment:
this is `IU` not `UI`. But that may not be the most obvious, could you use `mg` instead?